### PR TITLE
Fixes to PR #25656 - applies SME feedback

### PR DESCRIPTION
**Pull request by bobfuru**
Fixes https://github.com/openshift/openshift-docs/pull/25656 by applying feedback from @jlebon in that PR, which was already merged to master and CP'ed to 4.6. Once this is approved, it can also be merged so that it gets picked up by the review and merge work being done in https://github.com/openshift/openshift-docs/pull/25709.

### CODE REVIEW

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -150,16 +144,19 @@ a|Optional: Comma-separated indexes of partitions to preserve during the install
...
 
-a|`ignition.config.url`
+a|`coreos.inst.platform_id`
 
-a|Identifies the location of an Ignition config that you build to pass selected values to be used by the live installer. This argument can be used for adding disk partitioning or other features you want to have in place before the actual installation runs.
+a| Optional: The Ignition platform ID of the platform the {op-system} image is being installed on. Default is `metal`. This option determines whether or not to request an Ignition config from the cloud provider, such as VMware. For example: `coreos.inst.platform_id=vmware`.
+
+a|`ignition.config.url`
 
+a|Optional: The URL of the Ignition config for the live boot itself. For example, this can be used to customize how `coreos-installer` is invoked, or to run code before or after the install. This is different from `coreos.inst.ignition_url`, which is the Ignition config for the installed system.
```
Comment (ahardin-rh) [inline modules/installation-user-infra-machines-static-network.adoc:159]: s/live boot itself/live boot.
s/install/installation

File: modules/installation-user-infra-machines-static-network.adoc [hunk 2]
```diff
@@ -122,23 +122,17 @@ a|Required. The block device on the system to install to. It is recommended to u
...
 * If you are using `coreos.inst.image_url`, you must also use `coreos.inst.insecure`. This is because the bare-metal media are not GPG-signed for {product-title}.
 
 a|`coreos.inst.ignition_url`
 
-a|Optional: The URL of the Ignition config. If no URL is specified, no Ignition config will be embedded.
-
-a|`coreos.inst.platform_id`
-
-a| Optional: The Ignition platform ID of the platform the {op-system} image is being installed on. Default is bare metal. This option determines whether or not to request an Ignition config from the cloud provider, such as AWS or WMware. For example: `coreos.inst.platform_id=vmware`.
+a|Optional: The URL of the Ignition config to embed into the installed system. If no URL is specified, no Ignition config will be embedded.
```
Comment (ahardin-rh) [inline modules/installation-user-infra-machines-static-network.adoc:135]: s/will be/is

**Comment by ahardin-rh** [comment]
Just some minor suggestions from me! :+1:

**Pull request merged**